.github/workflows/release.yml

```diff
@@ -2,6 +2,7 @@ name: Build Docker Image
 
 # Release on any git tag
 on:
+  pull_request:
```

Comment:
how about on tagging?

Reply:
It is on tagging, `pull_request` is for testing only.